### PR TITLE
122 use entries meta

### PR DESCRIPTION
**Pull request by brooklynbagel**
Addresses #122. 

Was able to test

- [x] lastContrib, could not confirm it works because the distribute page isn't loading on the test db. Getting a 403 error for the location API
- [x] lastSignup works
- [x] lastUpvote works
- [x] size counter works

### CODE REVIEW

**Comment by codebling** [comment]
Hmm distribute seems to be working on test for me.  I'll have a look at this branch

**Comment by codebling** [comment]
Code looks good to me but I don't count yet so someone more qualified should review. 

> Hmm distribute seems to be working on test for me. I'll have a look at this branch

Yup ok can confirm it's not working on latest master. I'm going to try to see why

**Comment by codebling** [comment]
Ok, think I found the problem. If the IP geolocation fails, we never make a call to the DB. 

I didn't have the Ipdata API key in my `.env`
```
REACT_APP_IPDATA_KEY=f3af9a8dd3d807ac002984214c2ec3bf2377b84e215cf500bdfed261
```

This fixes the distribute page for me in dev mode on latest master. Maybe this is all that's missing on your end, as well?

Think we should always make the DB call to get the entries, even if the call to IPData fails, otherwise Leveler is more or less useable if: 1) IPData is down, 2) the user has an IP address which makes IPData choke, 3) we exceed the IPData API limit

I can throw together a quick PR for this if you all agree. 

(sidenote: sorry for honing in on small detail not even related to your PR and making the whole comment thread about it)

**Comment by brooklynbagel** [comment]
@codebling thanks for sleuthing around, much appreciated! (No worries for honing in on a side issue, it's better we fix things sooner rather than later). Anyway, providing the API let me check off that last item, so thanks again.

Please open an PR, I agree one is in order and the site shouldn't break down because of some issue with the location API. I'd recommend you either also open an issue or use the PR itself to discuss what's the best way to handle the case where the location API is not available. IMO the obvious answer is to just not filter based off location but there may be better ideas.

File: functions/entries-counter.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports.incrementCounter = (_snap, _context) => {
 
 // decrement collection size
 module.exports.decrementCounter = (_snap, _context) => {
-  entriesCounter
+  entrisMeta
```
Comment (samyoungnyc) [inline functions/entries-counter.js:18]: typo -> change to `entriesMeta`

Comment (brooklynbagel) [reply functions/entries-counter.js]: Fixed and rebased

**Pull request merged**